cpp/src/gandiva/function_registry_string.cc

```diff
@@ -209,6 +209,71 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
                      "convert_replace_invalid_fromUTF8_binary",
                      NativeFunction::kNeedsContext),
 
+      NativeFunction("convert_toDOUBLE", {"convert_todouble"}, DataTypeVector{float64()},
+                     binary(), kResultNullIfNull, "convert_toDOUBLE_binary",
+                     NativeFunction::kNeedsContext),
+
+      NativeFunction("convert_toDOUBLE_BE", {"convert_todouble_be"},
```

Comment:
function name should not be case sensitive..so no need to having same name in lower case as alias

Reply:
Changes applied and aliases removed